app/gui2/src/components/GraphEditor/NodeWidgetTree.vue

```diff
@@ -15,10 +15,12 @@ const props = defineProps<{
   icon: Icon
   connectedSelfArgumentId: Ast.AstId | undefined
   potentialSelfArgumentId: Ast.AstId | undefined
+  conditionalPorts: Set<Ast.AstId>
```

Comment:
Here also you could mention what exactly are conditions for "conditional ports".